src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,41 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+public class Tagging {
+	public enum TagAction {ADD, REMOVE};
+	
+	private ReadOnlyPerson person;
+	private UniqueTagList tag;
+	private TagAction tagAction;
+	
+	public Tagging(ReadOnlyPerson person, UniqueTagList uniqueTagList, TagAction tagAction) {
```

Comment:
The `Tagging` class should be association between `Person` and `Tag` rather than `ReadOnlyPerson` and `UniqueTagList`